config/settings.edn

```diff
@@ -1,9 +1,8 @@
 {:api-endpoint "http://www.getembed.com/4/" ;; REQUIRED - the api endpoint you wish to query
- :programme-id "" ;; REQUIRED - a Hecuba programme id
  :entity-type "Weather Station" ;; REQUIRED - the entity type you are searching for
  :entity-action "pullweather" ;; action request, sent in kafka message
- :zookeeper "localhost:2181" ;; REQUIRED - Zookeeper address:port
- :kafka-brokerlist "localhost:9092" ;; REQUIRED - Kafka address:port
- :kafka-topic "test-topic" ;; REQUIRED - Kafka topic to publish to.
+ :zookeeper "marathon.mesos:2181" ;; REQUIRED - Zookeeper address:port
+ :kafka-brokerlist "172.20.10.31:1025" ;; REQUIRED - Kafka address:port
```

Comment:
also, worth it having a look at aero which is pretty nice for having configs of several envs